packages/analytics/metric-cards/src/types/metric-card.ts

```diff
@@ -24,6 +25,6 @@ export interface MetricCardContainerOptions {
   fallbackDisplayText: string
   hasTrendAccess: boolean
   loading: boolean
-  cardSize?: string
```

Comment:
Should individual cards even have cardSize?

Reply:
Oh good point, this was copy/pasta from khcp-ui 😬 . Will remove.

Reply:
The cardSize prop is drilled down from the parent MetricCardContainer. So I should keep it here as well, no?

Reply:
I'll keep cardSize on MetricCardContainerOptions, and drop it on MetricCardDef